src/ess/reduce/time_of_flight/to_events.py

```diff
@@ -54,6 +54,8 @@ def to_events(
         nans = np.isnan(low) | np.isnan(high)
         low = np.where(nans, 0.0, low)
         high = np.where(nans, 0.0, high)
```

Comment:
Had trouble understanding the logic around the change. Should `low` and `high` actually be `left` and `right` (bin edge), until the added lines?

Reply:
It should probably be left and right everywhere...